pkg/lockfile/parse.go

```diff
@@ -8,6 +8,10 @@ import (
 	"strings"
 )
 
+// Represents when a package name could not be determined while parsing.
+// Currently, parsers are expected to omit such packages from their results.
+const unknownPkgName = "<unknown>"
```

Comment:
Can you explain a bit more about why this is required?

Reply:
Hello, if you refer to the string in itself , it is only used to specify that the value has been searched and not found.
From a technical point of view also empty string "" can be used if preferred.
For the `const`, it was necessary to use it solve a linter error (goconst). The string `<unknown>` is already used in apk parser code. Adding dpkg causes the linter to see multiple instances of the same constant string and thus it suggests using a const.
I can add my previously removed comment `// Avoid linter error (goconst) on some parsers` to make it clear
This const can be used also on apk parser to standardize the approach.